### DISCUSSION TITLE
Make a more powerful of H1,H2,H3 button

### DISCUSSION THREAD

**Discussion by ghost** [gsantner/markor#1723] [Ideas] [answered]
#### I always use H4 as the text of the markdown because when I use markdown to write homework, the H4 is proper to label the question number. And teacher can jump to it by toc.
I think that H1,H2,H3 can have more flexible function that if you clicking H1 first time will get H1, the clicking the second time will get H4, and click the third time will delete the label.  And H2 and H3 button is  the similar.
This will have more flexibility for the buttons.

**Comment by gsantner** [comment, marked as answer]
It sounds like a nice idea, but at same time might result in bad usage. Like you need to press 4 times a button to get a H4, instead of pressing a H4 button one time 😄 .  My opinion just is that we don't need more magic here

**Comment by harshad1** [comment]
We have snippet support now. Just create a snippet for H4. Long press the snippet button to insert the most recently inserted snippet.
